tests/cupy_tests/core_tests/test_ndarray.py

```diff
@@ -388,20 +387,22 @@ def test_cuda_array_interface_stream(self):
                 assert iface['stream'] == stream.ptr
 
 
-@pytest.mark.skipif(not cupy.cuda.runtime.is_hip,
-                    reason='This is supported on CUDA')
-class TestNdarrayCudaInterfaceNoneCUDA(unittest.TestCase):
+class TestNdarrayCudaInterfaceAttr(unittest.TestCase):
```

Comment:
I think it's ok to remove this entire test class. It does not make sense as `__cuda_array_interface__` no longer raises `AttributeError`.